src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.spec.jsx

```diff
@@ -42,6 +80,9 @@ describe('SelectTimecardPeriodType', () => {
       />
     );
 
+    // Wait a little bit longer
+    await waitFor(() => {});
```

Comment:
Not sure what are we waiting for here...

Reply:
Removing this waitFor(), causes the test to fail with the following message:
`Unable to find an element with the text: You must select a time period. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.`
I seems that the retrieval of the error message element is attempted before its rendering is complete

Reply:
ah ok - I thought you would usually waitFor('something'). Will have a look into it

Reply:
Change looks good to me.